themes/nord.json

```
@@ -0,0 +1,22 @@
+{
+    "author": {
+        "name": "Yavko",
```

```suggestion
        "name": "Arctic Ice Studio & Sven Greb",
```

Attribution should go to the original theme creator. In this case, it's Arctic Ice Studio & Sven Greb.

---

oops didnt know